api/resources_portal/test/dev_data/grants.json

```diff
@@ -0,0 +1,87 @@
+"grants": {
+    "0": {
+      "id": "1",
+      "created_at": "2020-01-30 16:12:25-07",
+      "updated_at": "2020-03-30 16:12:25-07",
+      "title": "Grant for Research in Melanoma Treatment",
+      "funder_id": "1"
+    },
+    "1": {
+      "id": "2",
+      "created_at": "2020-01-30 16:12:25-07",
+      "updated_at": "2020-03-30 16:12:25-07",
+      "title": "Grant for Research in Tumor Growth",
+      "funder_id": "3"
+    }
+  },
+  "grants_materials": {
+    "0": {
+      "id": "1",
+      "grant_id": "1",
+      "material_id": "1"
+    },
+    "1": {
+      "id": "2",
+      "grant_id": "1",
+      "material_id": "2"
+    },
+    "2": {
+      "id": "3",
+      "grant_id": "1",
+      "material_id": "3"
+    },
+    "3": {
+      "id": "4",
+      "grant_id": "1",
+      "material_id": "4"
+    },
+    "4": {
+      "id": "5",
+      "grant_id": "2",
+      "material_id": "5"
+    },
+    "5": {
+      "id": "6",
+      "grant_id": "2",
+      "material_id": "6"
+    },
+    "6": {
+      "id": "7",
+      "grant_id": "2",
+      "material_id": "7"
+    },
+    "7": {
+      "id": "7",
+      "grant_id": "2",
+      "material_id": "8"
+    }
+  },
+  "grants_organizations": {
+    "0": {
+      "id": "1",
+      "grant_id": "1",
+      "organization_id": "1"
+    },
+    "1": {
+      "id": "2",
+      "grant_id": "2",
+      "organization_id": "4"
+    }
+  },
+  "grants_users": {
+    "0": {
+      "id": "1",
+      "grant_id": "1",
+      "user_id": "1"
+    },
+    "1": {
+      "id": "1",
+      "grant_id": "1",
+      "user_id": "2"
```

Comment:
Why is the PostDoc on the grant as well?

Reply:
It was meant to exhibit the co-ownership that grants can have, but I forgot to reflect it in the README. I'll add it there.

Reply:
Hmm, I think the situation we want to simulate for co-ownership would be two PI's who write a grant together and therefore both get credit for sharing resources produced by it.

Does that sound right to you @dvenprasad ?

Reply:
Yeah, we do want co-ownership to be between two PIs.

Reply:
Ok. I'll change it so that PrimaryProf and SecondaryProf both share the grant.